action.yml

```diff
@@ -59,6 +59,11 @@ inputs:
     required: false
     default: ''
 
+  branch_name:
```

Comment:
Can we name this `branch` to stay consistent with the underlying CLI flag? 🙇🏼

Reply:
I was a bit back and forth on this one and decided on: `More explicit better`. Though I don't feel strongly about this, I'll change it. 👍